skorch/callbacks/lr_scheduler.py

```diff
@@ -56,16 +57,37 @@ def on_epoch_begin(self, net, **kwargs):
         else:
             self._lr_scheduler.step(epoch)
 
+    def on_batch_begin(self, net, **kwargs):
+        if isinstance(self._lr_scheduler, CyclicLR):
```

Comment:
Maybe `hasattr(self._lr_scheduler, 'batch_step') and callable(self._lr_scheduler.batch_step)` would be better suited here?